pkg/controller/direct/networkmanagement/connectivitytest_mappers.go

```diff
@@ -33,9 +35,13 @@ func EndpointObservedState_FromProto(mapCtx *direct.MapContext, in *pb.Endpoint)
 	// MISSING: Port
 	// MISSING: Instance
 	// MISSING: ForwardingRule
-	out.ForwardingRuleTarget = direct.Enum_FromProto(mapCtx, in.GetForwardingRuleTarget())
+	if in.ForwardingRuleTarget != nil {
+		out.ForwardingRuleTarget = direct.ZeroBasedEnum_FromProto(mapCtx, in.GetForwardingRuleTarget())
```

Comment:
This means we will convert for `FORWARDING_RULE_TARGET_UNSPECIFIED`? Is this just to keep fuzz testing happy? Or this is actually a valid config.

Reply:
The value is a pointer, this means that when it is non-nil in the proto struct, it's probably intentionally set by the API.